src/networkcloud/HISTORY.rst

```diff
@@ -31,7 +31,7 @@ Release History
 
 2.0.0b7
 ++++++++
-* This version requires a minimum of 2.66 Azure core CLI. See release notes for more details: https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/docs-ref-conceptual/release-notes-azure-cli.md
+* This version requires a minimum of 2.66 Azure core CLI. See release notes for more details: https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/docs-ref-conceptual/Latest-version/release-notes-azure-cli.md
 * This version upgrades the internal generation tool aaz-dev-tools to 3.1.0. Refer to the release notes for more details: https://github.com/Azure/aaz-dev-tools/releases/tag/v3.1.0.
```

Comment:
If historyneeds to be updated, one more case in managednetworkfabric: https://github.com/Azure/azure-cli-extensions/blob/main/src/managednetworkfabric/HISTORY.rst?plain=1#L28